src/libraries/System.Formats.Tar/tests/TarFile/TarFile.ExtractToDirectory.Stream.Tests.cs

```diff
@@ -390,8 +397,13 @@ private void ExtractToDirectory_ExactRootDirMatch_Links_Throws(TarEntryFormat fo
             string linkTargetFileName = "file.txt";
             string linkFileName = "link";
 
+#if TARGET_MACCATALYST || TARGET_IOS || TARGET_TVOS
+            // the folder used to store files needs to have a shorter path on Apple mobile platforms,
+            // because the TempDirectory gets created in folder with a path longer than 100 bytes
+            using TempDirectory root = new TempDirectory("/Users/helix-runner/Library/Developer/CoreSimulator/Devices/tempDir");
```

Comment:
Do you have control over `tempDir`? Could it just be `tmp`, or `t`?

Reply:
It is an arbitrary name and probably we can set even shorter name.